dotnet/src/SemanticKernel/Orchestration/SKContextExtensions.cs

```diff
@@ -15,13 +15,20 @@ internal static class SKContextExtensions
     /// <param name="skillName">The skill name</param>
     /// <param name="functionName">The function name</param>
     /// <param name="registeredFunction">The registered function, if found</param>
-    internal static bool IsFunctionRegistered(this SKContext context, string skillName, string functionName, [NotNullWhen(true)] out ISKFunction? registeredFunction)
+    internal static bool IsFunctionRegistered(this SKContext context,
+        string skillName,
```

Comment:
Nit:
```suggestion
        string? skillName,
```
since skillName is allowed to be null per the below checks.